src/tests/system/tests/test_ipa.py

```diff
@@ -640,3 +641,874 @@ def test_ipa__switch_user_with_smartcard_authentication(client: Client, ipa: IPA
     assert (
         "ipacertuser1" in result.stdout
     ), f"'ipacertuser1' not found in 'whoami' output! Stdout content: {result.stdout}"
+
+
+@pytest.mark.importance("medium")
+@pytest.mark.topology(KnownTopology.IPA)
+def test_ipa__hbac_permitted_users_can_login(client: Client, ipa: IPA):
+    """
+    :title: Validate HBAC rule-based SSH access control using the `sshd` service
+
+    :setup:
+      1. Add users.
```

Comment:
Let's keep the docstring consistent within the file, remove the period at the end of each doc string setup, setp and expected result.

Reply:
updated.